### ISSUE TITLE
DryRun Feature for OPA bundle.

### ISSUE THREAD

**Issue by bj-1795** [open-policy-agent/opa#3883]
I have a usecase that we should do a dryrun of policies before enforcement on production cluster to make sure our policies are working as expected, is this feature available with OPA bundle or policy as a configmap.

**Comment by anderseknert** [comment]
Yes, using the Kubernetes API with the `dryRun` query parameter, or kubectl with `--dry-run=server` the object(s) will be sent off to the Kubernetes API server without being persisted, while still running validation checks like admission control. Your policy would normally not need to take this into account.
